packages/core/content-type-builder/server/services/content-types.js

```diff
@@ -19,7 +19,7 @@ const getRestrictRelationsTo = (contentType = {}) => {
     return ['oneWay', 'manyWay'];
   }
 
-  if (uid.startsWith(coreUids.PREFIX) || uid === pluginsUids.UPLOAD_FILE) {
+  if (uid.startsWith(coreUids.PREFIX) || Object.values(pluginsUids).includes(uid)) {
```

Comment:
I'm not sure if this is the best way to prevent relations to be done to certain content type. So let me know if there's a better way.

Reply:
I think using `pluginOptions.contentType: { visible: false }` should work for these content-types as we don't want them to appear not only remove them from the relations

Reply:
Do we want to have a global `{ visible: false }` or should we use the already existing `{ 'content-type-builder': { visible: false } }` to remove it from the relation list?